app/services/filters_service.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# This module encapsulates filter-related functionality
+module FiltersService
+  # Class method Public (self) called 'matching_params' that takes a 'params' parameter
```

Comment:
Coloca um comentário  `# @param params [Hash]` pra documentar o tipo do parâmetro
Muda: # `Class method Public (self) called 'matching_params' that takes a 'params' parameter` para algo mais descritivo como: método responsável por retornar os parâmetros de busca de fotógrafo que são válidos.